src/components/forms/TokenInput.tsx

```diff
@@ -44,25 +57,21 @@ const OptionsContainer = styled.div<{ open: boolean }>`
 
 const OptionContainer = styled.div<Pick<TokenOptionProps, 'selected'>>`
   display: flex;
+  flex-direction: row;
   align-items: center;
-  height: 46px;
   overflow-x: hidden;
-
   padding: ${({ theme }) => theme.spacing.xs};
-
+  box-sizing: border-box;
```

Comment:
Surprised this is necessary, I thought we had `* { box-sizing: border-box; }` set

Reply:
Ah, it might not be – will check